pkg/rest/define/config.go

```diff
@@ -0,0 +1,32 @@
+package define
```

Comment:
We call this `types` in crc, though if you are used to `define` this also works for me.